lib/services/chain_loader.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+import 'package:silentshard/screens/sign/sign_request_view_model.dart';
+
+class ChainLoader {
+  late List<dynamic> _chains;
+
+  ChainLoader() {
+    loadChains();
+  }
+
+  Future<void> loadChains() async {
+    var jsonString = await rootBundle.loadString("assets/json/chain.json");
+    _chains = json.decode(jsonString);
```

Comment:
Probably we should parse each entry as well, so that we don't need to do it on each approval again?